server/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -501,7 +501,7 @@ static OverLimitStrategy createOverLimitStrategy(boolean trackRealMemoryUsage) {
                 HierarchyCircuitBreakerService::realMemoryUsage,
                 createYoungGcCountSupplier(),
                 System::currentTimeMillis,
-                5000,
+                500,
```

Comment:
++ just adding my explicit agreement for this change in value ( from 5 secs to 1/2 sec )